drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -294,6 +335,51 @@ local function temp_attr_handler(driver, device, ib, response)
   end
 end
 
+local mired_bounds_handler_factory = function(minOrMax)
+  return function(driver, device, ib, response)
+    if ib.data.value ~= nil then
```

Comment:
Nit: Consider using an early return for readability.

Reply:
Updated as suggested. See 775564f06809491f7f3c5359f887bf7864edad48.